packages/task/src/browser/task-terminal-widget-manager.ts

```diff
@@ -117,40 +117,42 @@ export class TaskTerminalWidgetManager {
         });
     }
 
-    async open(options: TaskTerminalWidgetOpenerOptions): Promise<TerminalWidget> {
-        const dedicated = TaskTerminalWidgetOpenerOptions.isDedicatedTerminal(options);
-        if (dedicated && !options.taskConfig) {
+    async open(factoryOptions: TerminalWidgetFactoryOptions, openerOptions: TaskTerminalWidgetOpenerOptions): Promise<TerminalWidget> {
```

Comment:
This method was not released yes, so no need for breaking changes. Preemptive comment :)